src/vos/tests/vts_mvcc.c

```diff
@@ -955,14 +955,14 @@ static struct op operations[] = {
 	{"querymaxa",	T_R,	L_D,	L_NIL,	R_NE,	W_NIL,	querymaxa_f},
 	{"querymaxr",	T_R,	L_A,	L_NIL,	R_NE,	W_NIL,	querymaxr_f},
 	{"querymaxda",	T_R,	L_O,	L_NIL,	R_NE,	W_NIL,	querymaxda_f},
-	{"querymaxdr",	T_R,	L_A,	L_NIL,	R_NE,	W_NIL,	querymaxdr_f},
+	{"querymaxdr",	T_R,	L_O,	L_NIL,	R_NE,	W_NIL,	querymaxdr_f},
 	{"querymaxar",	T_R,	L_D,	L_NIL,	R_NE,	W_NIL,	querymaxar_f},
 	{"querymaxdar",	T_R,	L_O,	L_NIL,	R_NE,	W_NIL,	querymaxdar_f},
 	{"querymind",	T_R,	L_O,	L_NIL,	R_NE,	W_NIL,	querymind_f},
 	{"querymina",	T_R,	L_D,	L_NIL,	R_NE,	W_NIL,	querymina_f},
 	{"queryminr",	T_R,	L_A,	L_NIL,	R_NE,	W_NIL,	queryminr_f},
 	{"queryminda",	T_R,	L_O,	L_NIL,	R_NE,	W_NIL,	queryminda_f},
-	{"querymindr",	T_R,	L_A,	L_NIL,	R_NE,	W_NIL,	querymindr_f},
+	{"querymindr",	T_R,	L_O,	L_NIL,	R_NE,	W_NIL,	querymindr_f},
```

Comment:
I think this introduces logically invalid reads since the akey path will blindly use path[3].   We need to add some handling for this if this is the direction we go

Reply:
Setting the akey directly to an arbitrary key without using set_akey or path should solve the problem.